testing/src/test/java/io/stargate/it/http/graphql/graphfirst/FilesResourceTest.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphfirst;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.graphql.GraphqlTestBase;
+import java.io.IOException;
+import java.util.Map;
+import javax.ws.rs.core.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class FilesResourceTest extends GraphqlTestBase {
+  private static String DEPLOYED_SCHEMA_VERSION;
+
+  @BeforeAll
+  public static void deploySchema(@TestKeyspace CqlIdentifier keyspaceId) {
+    // deploySchema
+    Map<String, Object> response =
+        executeGraphqlAdminQuery(createSchema(keyspaceId), "deploySchema");
+    assertThat(response).isNotNull();
+    DEPLOYED_SCHEMA_VERSION = getDeployedVersion(response);
+    assertThat(DEPLOYED_SCHEMA_VERSION).isNotNull();
+  }
+
+  @Test
+  public void shouldGetGraphQLDirectivesFile() throws IOException {
+    // given
+    OkHttpClient httpClient = getHttpClient();
+    String url = String.format("%s:8080/graphqlv2/files/cql_directives.graphql", host);
+    Request getRequest =
+        new Request.Builder()
+            .get()
+            .addHeader("content-type", MediaType.TEXT_PLAIN)
+            .url(url)
+            .build();
+
+    // when
+    Response response = httpClient.newCall(getRequest).execute();
+
+    // then
+    assertThat(response.code()).isEqualTo(200);
+    assertThat(response.body()).isNotNull();
+    String responseBody = response.body().string();
+    assertThat(responseBody).isNotEmpty();
```

Comment:
RestUtils does this already
https://github.com/stargate/stargate/blob/master/testing/src/test/java/io/stargate/it/http/RestUtils.java#L45-L64